src/Parser/DeathKnight/Blood/Modules/Features/MarrowrendUsage.js

```diff
@@ -44,6 +46,7 @@ class MarrowrendUsage extends Analyzer {
 
     if(this.selectedCombatant.hasTrait(SPELLS.BONES_OF_THE_DAMNED.id)) {
       this.hasBonesOfTheDamned = true;
```

Comment:
I don't think hasBonesOfTheDamned is needed.  I don't see another usage of it other than setting it to true.

Reply:
It is, kind of. 
https://github.com/WoWAnalyzer/WoWAnalyzer/blob/463776f08f4514ba4a4f8212fba9aa17965dee67/src/Parser/DeathKnight/Blood/Modules/Features/MarrowrendUsage.js#L104

I'm using it to determinate if I want to hightlight a cast in the timeline or not (in case that `this.currentBoneShieldBuffer` returns a value higher than 3 and the player has no BotD). Could technically remove that value and make it dependant on `this.refreshAtStacks`, but I'd be less readable.